fdbclient/ManagementAPI.actor.cpp

```diff
@@ -1080,7 +1080,10 @@ ACTOR Future<CoordinatorsResult> changeQuorum(Database cx, Reference<IQuorumChan
 			choose {
 				when( wait( waitForAll( leaderServers ) ) ) {}
```

Comment:
Maybe you should keep the COORDINATOR_UNREACHABLE below, and instead wait on `quorum(leaderServers, (leaderServers.size()+1)/2)` if force is true here?